exercise/templates/exercise/staff/_submissions_table.html

```diff
@@ -61,7 +61,7 @@
             <td>
                 {% profiles submission.submitters.all instance is_teacher %}
             </td>
-            <td>
+            <td datetime="{{ submission.submission_time|date:'Y-m-d H:i:s' }}">
```

Comment:
Is `datetime` supposed to be a data attribute `data-datetime`? Is it not invalid HTML to use custom non-standard attributes? Even if this works, a data attribute would be the correct approach according to the web standards, right?

Reply:
Yep. This needs to be  prefixed with `data-`.

Reply:
Aah so that is why the `data-` prefix is used, did not know the reason for using data attributes.

I assume we do not want to use the <time> tag that has its own datetime attribute (which I did now know about when I used my custom datetime attribute).

Reply:
The data attribute is fine. The `<time>` tag is new in HTML5 and it would fit semantically to this usecase, but it is not supported by IE and offers no real benefit over the data attribute. Note that jQuery can read data attributes with the `attr()` method and also the special `data()` method that does not take the data prefix in the attribute name. You can also change the stored data without modifying the actual attribute at runtime, in which case jQuery stores the data key-value pairs elsewhere in memory.

https://api.jquery.com/data/